release_draft.md

```diff
@@ -0,0 +1,299 @@
+(nothing is modified) in order to ensure that the correct arguments are being used to trigger a particular release.
+Ignition releasers should **always** call `release.py` with `--dry-run` first in order to ensure that proper commands are being used to trigger releases.
+
+The script needs to be run from the repository with the source code (i.e., the repository where the Ignition library version bump pull request took place):
+
+```bash
+# Example of dry-run for ign-cmake2 bumped to 2.0.1
+cd ign-cmake2
+git checkout ign-cmake2
+~/release-tools/release-repo-scripts/release.py ign-cmake2 2.0.1 <test_token> --dry-run
+```
+
+#### release.py for stable releases
+
+```bash
+# Ignition library named foo and major version X
+cd ign-fooX
+git checkout ign-fooX
+
+# Example ign-cmake2 bumped to 2.0.1 with test_token credential
+cd ign-cmake2
+git checkout ign-cmake2
+# please replace <test_token> with real release token (check crendentials section)
+~/release-tools/release-repo-scripts/release.py ign-cmake2 2.0.1 <test_token>
+```
+
+#### release.py for prereleases or nightly
+
+When releasing prereleases or nightly releases, there are some special flags to
+be set. The `--upload-to-repo` argument is mandatory when running `release.py`, and
+should be set to `prerelease` or `nightly`.
+
+```bash
+# Example ign-cmake2 bumped to prerelease 2.0.0~pre1 with test_token credential
+cd ign-cmake2
+git checkout ign-cmake2
+# please replace <test_token> with real release token (check crendentials section)
+~/release-tools/release-repo-scripts/release.py ign-cmake2 2.0.0~pre1 test_token --upload-to-repo prerelease
+```
+Nightly invocation is generally coded in the server. The version will be taken from
+the last changelog entry and [modified during building](releasing/versioning_pre_nightly).
+No source code will be uploaded, but taken directly in the binary build from
+`--nightly-src-branch`.
+
+```bash
+# Example ign-cmake2 nightly from main branch with test_token credential
+cd ign-cmake2
+git checkout ign-cmake2
+# please replace <test_token> with real release token (check crendentials section)
+~/release-tools/release-repo-scripts/release.py ign-cmake2 2.0.0~pre1 test_token --upload-to-repo nightly --nightly-src-branch main
+```
+#### release.py for revision bumps
+
+Bumping the [revision number for binary packages](#versioning) is a special case of releasing
+since the original tarball with the source code will remain the same. Once the
+release repository is ready with the new release version, `release.py` needs the
+`--only-bump-revision-linux` flag:
+
+```bash
+# Example ign-cmake2 bumped from 2.0.1-1 to 2.0.1-2 with test_token credential
+cd ign-cmake2
+git checkout ign-cmake2
+# please replace <test_token> with real release token (check crendentials section)
+~/release-tools/release-repo-scripts/release.py ign-cmake2 2.0.1 test_token --only-bump-revision-linux -release-version 2
+```
```

Comment:
after the `release.py` example sections, should we add some information about how to view the status of the builds that were triggered so that you can make sure everything succeeds? Currently, we have some information about this [here](https://github.com/ignition-tooling/release-tools/blob/a566144052ea70964f4b77cc34886395c50286f0/README.md) (point 10 at the end of the document - starts with "check that:")

Reply:
Yeah I think a checklist would be best

Reply:
I added the previous info in 5061539f5c4e0435b78fd07af1b81ec077ceadce